doc/cascadia/profiles.schema.json

```diff
@@ -19,7 +19,7 @@
       "type": "string"
     },
     "ProfileGuid": {
-      "default": "{}",
+      "default": "{00000000-0000-0000-0000-000000000000}",
```

Comment:
this is not the default guid; there is no default guid; using this as a default will make it so people actually put it in their profile (because default is "vscode's automatic suggestion"), and then they'll ask us why their profiles don't work properly.

Reply:
the default here intentionally breaks with the validation regex so it flags as an error

Reply:
So, this is more something I more wanted to propose more as a jumping off point. I get that there's no default guid, because it's required that the user defines it. But it feels a little weird to put in the description, "it follows this format" and then just give them a blank `{}`. I was hoping this would make the format more clear.

More just thinking of miscellaneous ways we can make the schema better but I'd be ok removing this one. 😊

Also, this doesn't break the regex pattern. I'm not getting an error from the schema (even when I delete it as default).